### DISCUSSION TITLE
Add OCI regions : Singapore 2, Riyadh, Jovanovac

### DISCUSSION THREAD

**Discussion by samsit88** [VarunAgw/CloudPingTest#49] [Ideas] [answered]
Could you add the following Oracle OCI regions ?

- ap-singapore-2 (XSP)
- me-riyadh-1 (RUH)
- eu-jovanovac-1 (BEG)

Thx CloudPingTest !!

**Comment by VarunAgw** [comment, marked as answer]
Added - https://cloudpingtest.com/oracle
